taskcat/_cli_modules/test.py

```diff
@@ -100,6 +115,8 @@ def run(  # noqa: C901
         enable_sig_v2: bool = False,
         keep_failed: bool = False,
         output_directory: str = "./taskcat_outputs",
+        minimal_output: bool = False,
+        dont_wait_for_delete: bool = True,
```

Comment:
Please set this to False by default.

Reply:
oops, that was my intent, confused myself with the double negative